example/lib/screens/user_screen.dart

```diff
@@ -304,20 +305,18 @@ class Home extends StatelessWidget {
   }
 
   getAppToWebSingleSignOn(BuildContext context) async {
-    var oneginiAppToWebSingleSignOn = await Onegini.instance.userClient
-        .getAppToWebSingleSignOn(
-            "https://login-mobile.test.onegini.com/personal/dashboard")
-        .catchError((error) {
-      if (error is PlatformException) {
-        showFlutterToast(error.message);
-      }
-    });
-    if (oneginiAppToWebSingleSignOn != null) {
+    try {
+      final oneginiAppToWebSingleSignOn = await Onegini.instance.userClient
+          .getAppToWebSingleSignOn(
+              "https://login-mobile.test.onegini.com/personal/dashboard");
+      // FIXME: Find a different way to launch this.
```

Comment:
Please create a ticket and add a number here, so we don't forget about this

Reply:
Fixed it in this PR as the library is first party from Flutter and it's still supported, just not that specific api that we were using.